packages/babel-plugin-debug-ids/src/index.test.ts

```diff
@@ -230,6 +230,19 @@ describe('babel plugin', () => {
     `);
   });
 
+  it('should handle createViewTransition assigned to const', () => {
+    const source = `
+      import { createViewTransition } from '@vanilla-extract/css';
+
+      const myViewTransition = createViewTransition("title");
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { createViewTransition } from '@vanilla-extract/css';
+      const myViewTransition = createContainer("title", "myViewTransition");
```

Comment:
This will get picked up by tests.

```suggestion
      const myViewTransition = createViewTransition("title", "myViewTransition");
```